lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/test/RoundTripTest.java

```diff
@@ -48,7 +49,8 @@ public void testRoundTrip() throws Exception {
   public void testRoundTripWithClassHierarchy() throws Exception {
     // TODO: use 'getConfiguration' instead of 'getConfigurationWithoutList' after #192 is fixed
     final Configuration confBefore = ObjectTreeTest.getConfigurationWithoutList();
-    final ClassHierarchy c = new ProtocolBufferClassHierarchy(ProtocolBufferClassHierarchy.serialize(confBefore.getClassHierarchy()));
+    final ClassHierarchySerializer chSerializer = new AvroClassHierarchySerializer();
+    final ClassHierarchy c = chSerializer.fromString(chSerializer.toString(confBefore.getClassHierarchy()));
```

Comment:
In AvroConfigurationSerializer, both fromAvro and toAvro are public. As the generated Avro classes like AvroNode is public, we don't need to hide it in our method I guess. And sometime, the client may want to work on those Avro classes directly.